lib/assessment.js

```diff
@@ -8,38 +8,64 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue) {
+  return previousValue < currentValue ? previousValue : currentValue;
 };
-
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
+var sum = [[0,1,2,3].reduce(function sum(a,b) {
+return a + b})];
```

Comment:
The function passed as an argument to `reduce`:

```js
function sum(a,b) {
    return a + b
}
```

is a correct answer to this question. However, the variable `sum` stores a number, which is of the wrong type to be an answer to this question.

1